Cookbook/Technical-Documents/Testing.md

```diff
@@ -0,0 +1,25 @@
+## Testing
+
+Our team is composed by multiple squads which are responsible for specific parts of the product. Based on that, each squad is responsible for owning and maintaining their own testing, from unit tests to UI Automation tests, meaning that any failure should be investigated and fixed by the squad responsible. If for some reason the squad is unavailable, e.g. holidays, then one of the release engineers should take ownernship of the failure when we are in the process of release a new version of that app otherwise it should fallback to the Support Engineer.
+
+### Execution
+
+We use [fastlane](https://fastlane.tools) to run most of our common and frequent tasks and UI Automation tests are no exception, we have specific lanes defined for each part of the product and they can be triggered on demand, either by running them locally or on CI from the #ios-build channel.
+
+```
+# Trigger a lane to be executed locally from the terminal
+$ bundle exec fastlane {lane}
+
+# Trigger a lane to be executed on CI from the #ios-build channel
+/fastlane {lane} branch:{branch}
+```
+
+Listing all lanes available can be done with `bundle exec fastlane list` in the terminal.
+
```

Comment:
We might want to mention that UI Test lanes all start with `ui_test_*` and that the `ui_test_babylon` lane is kinda special, allowing to test a specific feature on demand using `fastlane ui_test_babylon feature:MyAwesomeFeature` – using the XCTest class name for the `feature:` parameter (related: https://github.com/Babylonpartners/babylon-ios/pull/8760)